library/src/main/java/de/agilecoders/wicket/webjars/util/RecentVersionCallable.java

```diff
@@ -47,10 +45,8 @@ public String call() throws Exception {
      * @return recent version string
      */
     private static String collectRecentVersionFor(final String partialPath) {
-        return Holder.recentVersionProvider.findRecentVersionFor(partialPath);
+        AssetsMap assetsMap = AssetsMap.get();
+        return assetsMap.findRecentVersionFor(partialPath);
     }
 
-    static final class Holder {
```

Comment:
Do you unload the ClassLoader that loaded wicket-webjars ?
Is this the web container application class loader or you use more fine grained hierarchy of class loaders ?